rewards/df-intro.md

```diff
@@ -52,6 +52,15 @@ Rewards are paid in OCEAN to contributors actively working to improve Ocean Prot
 | --------------- | -------------- |
 | 5,000 OCEAN     | 70,000 OCEAN    |
 
+**November 2** will then change when Predictoor DF is launched:
```

Comment:
Change: "November 2" --> "Nov 9"

Why:
- The blog initially got the date wrong. Correct date is Nov 9
- Wherever you can reduce the # characters while retaining clarity, you should. Hence "November" -> "Nov".

Reply:
Done